dspace/config/crosswalks/DIM2DATACITE.xsl

```diff
@@ -322,7 +322,7 @@
 				<descriptions>
 					<description descriptionType="Other">
 					        <!-- Exclude provenance fields, since these often contain private information -->
-						<xsl:value-of select="dspace:field[@element='description' and not(@qualifier='provenance')]"/>
+						<xsl:value-of select="dspace:field[@element='description']"/>
```

Comment:
Why this change? We don't want to send provenance metadata (such as rejection messages) to DataCite.

Reply:
It doesn't; everything in the metadata.hide area of the config is not sent to DataCite. That was the correct way to do it, not this other way.

Reply:
If that isn't the case (I thought I'd tested it adequately), we can roll this one change back out, but this is a band-aid on a bigger bug if so.

Reply:
I will test -- The documentation in the config indicates that the "hide" settings only apply to the DSpace GUI, not to the DataCite process. And there may be other command-line tools that did the XSL transform and sent content to DataCite without running through the full DSpace process.

Reply:
Okay. It is easy enough for me to roll out that commit from the crosswalk.